src/components/Person/Person.jsx

```diff
@@ -1 +1,40 @@
-// export const Person = ({ person }) => ();
+/* eslint-disable react/jsx-one-expression-per-line */
+export const Person = ({ person }) => {
+  if (person.sex === 'f') {
+    return (
+      <section className="Person">
+        <h2 className="Person__name">My name is {person.name}</h2>
+        {typeof (person.age) === 'number'
```

Comment:
don't need to check if age is type number, think it is redundant here so you can remove this ternary operator